R/stat-density.r

```diff
@@ -21,6 +21,8 @@
 #'   \item{count}{density * number of points - useful for stacked density
 #'      plots}
 #'   \item{scaled}{density estimate, scaled to maximum of 1}
+#'   \item{ndensity}{alias for stat(scaled), to mirror the syntax of
+#'    geom_bin}
```

Comment:
Can you put `stat(scaled)` and `geom_bin` in backticks so they show up as code? And also please write `geom_bin()` (with parentheses).